### ISSUE TITLE
Question: Are there any ports of this library to Java/Kotlin?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [adamian98/pulse#82]
As the title says.

I'm looking for a nice "image-enhancing" library that given an image and a target resolution could create an enhanced version of this image (having the new resolution).

I wish to use it on Android, and I don't think Python can be used so easily for Android.
If I'm wrong, please let me know how to do it on Android.

**Comment by sunatte-saad** [comment]
not possible on a phone, as it requires GPU to run. The possible way to do it is to run it a server having GPU and then create API's to interact with it using android

**Comment by AndroidDeveloperLB** [comment]
A phone has GPU, just on the same chip...
Are you saying it requires a lot of computing power?
What if I wish to check it out, to see if maybe the time it takes is ok?
If I want to enlarge some small images to about x3 of their sizes, for example...

**Comment by sunatte-saad** [comment]
Yes it takes lots of computing power. Also the architecture matters.
If you just want to check it out, you may use Google colab

**Comment by AndroidDeveloperLB** [comment]
I meant to check out on Android
